### PR TITLE
HPCC-18883 Document ping_min_ver ESDL attribute

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>
@rpastrana please review. Pdf avail , linkbelow

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:

http://10.240.32.243/view/Docs%207.0/job/JimDocBuild7/CE=ubuntu-14.04-amd64-1/lastSuccessfulBuild/artifact/build-docs/Release/docs/EN_US/ESDL_LangRef_EN_US-7.0.0-trunk0.pdf

### CODE REVIEW

**Comment by rpastrana** [comment]
We should mention that this attribute only makes sense in the service definition. Also, we should mention that ping() is automatically generated, therefore if a user wants to suppress, they can use this attribute.

**Comment by JamesDeFabia** [comment]
@rpastrana Edits made, back to you

**Comment by JamesDeFabia** [comment]
@rpastrana Please give it another look

File: docs/EN_US/DynamicESDL/DESDL-Mods/ESDLping_min_ver.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect1 id="ESDL_Attributes_ping_min_ver">
+  <title>ping_min_ver</title>
+
+  <para>Dynamic ESDL services automatically add a ping method for testing and
```
Comment (rpastrana) [inline docs/EN_US/DynamicESDL/DESDL-Mods/ESDLping_min_ver.xml]: it's not necessarily for testing/development, it's more for operations/monitoring.

**Review by rpastrana — COMMENTED** [review]
@JamesDeFabia one comment

**Review by rpastrana — APPROVED** [review]
@JamesDeFabia

**Comment by JamesDeFabia** [comment]
@richardkchapman This should be good to merge now

**Pull request merged**